[OCPBUGS-5740]: Adding step for single-node OpenShift (#54796)
openshift/openshift-docs: modules/restore-replace-crashlooping-etcd-member.adoc

[source,terminal]
----
$ oc get etcd/cluster -oyaml

> =======
> . If you are using {sno}, restart the node. Otherwise, you might encounter the following error in the etcd cluster Operator:
> +
> .Example output
> [source,terminal]
> ----
> EtcdCertSignerControllerDegraded: [Operation cannot be fulfilled on secrets "etcd-peer-sno-0": the object has been modified; please apply your changes to the latest version and try again, Operation cannot be fulfilled on secrets "etcd-serving-sno-0": the object has been modified; please apply your changes to the latest version and try again, Operation cannot be fulfilled on secrets "etcd-serving-metrics-sno-0": the object has been modified; please apply your changes to the latest version and try again]
> ----

Reviewer: Noticed the build failure. Is this what you're looking to do?

```suggestion
----

. If you are using {sno}, restart the node. Otherwise, you might encounter the following error in the etcd cluster Operator:
+
.Example output
[source,terminal]
----
EtcdCertSignerControllerDegraded: [Operation cannot be fulfilled on secrets "etcd-peer-sno-0": the object has been modified; please apply your changes to the latest version and try again, Operation cannot be fulfilled on secrets "etcd-serving-sno-0": the object has been modified; please apply your changes to the latest version and try again, Operation cannot be fulfilled on secrets "etcd-serving-metrics-sno-0": the object has been modified; please apply your changes to the latest version and try again]
----
```

[source,terminal]

> ----
> $ oc get etcd/cluster -oyaml
> =======

Reviewer: ```suggestion
----
$ oc get etcd/cluster -oyaml
----
```

https://10.0.154.204:2379 is healthy: successfully committed proposal: took = 16.698331ms
https://10.0.164.97:2379 is healthy: successfully committed proposal: took = 16.621645ms
----

.Next steps

> If you use single-node {product-title}, restart the node. Otherwise, you might encounter the following error in the etcd cluster Operator:

Reviewer 2: ```suggestion
If you are using {sno}, restart the node. Otherwise, you might encounter the following error in the etcd cluster Operator:
```

Reviewer 2: similar comment for the other two files

Author: Good catch!
